frontend/src/utilities/useDraggableTable.ts

```diff
@@ -29,12 +29,13 @@ const useDraggableTable = (
   const bodyRef = React.useRef<HTMLTableSectionElement>(null);
 
   const onDragStart: TrProps['onDragStart'] = (evt) => {
-    evt.dataTransfer.effectAllowed = 'move';
-    evt.dataTransfer.setData('text/plain', evt.currentTarget.id);
-    const draggedItemId = evt.currentTarget.id;
+    const updatedEvt = { ...evt };
+    updatedEvt.dataTransfer.effectAllowed = 'move';
+    updatedEvt.dataTransfer.setData('text/plain', updatedEvt.currentTarget.id);
+    const draggedItemId = updatedEvt.currentTarget.id;
 
-    evt.currentTarget.classList.add(styles.modifiers.ghostRow);
-    evt.currentTarget.setAttribute('aria-pressed', 'true');
+    updatedEvt.currentTarget.classList.add(styles.modifiers.ghostRow);
+    updatedEvt.currentTarget.setAttribute('aria-pressed', 'true');
```

Comment:
Drag events work this way and must be modified.

Best to add a new value to the rule option support ignoring the rule.
"ignorePropertyModificationsForRegex": ["^assignable[A-Z]"]

Then rename `evt` to `assignableEvent`.